### DISCUSSION TITLE
[Feature Request] Stared & Liked repos

### DISCUSSION THREAD

**Discussion by mikumikudice** [community/community#2981] [Other Feature Feedback, Questions, & Ideas]
I believe that adding a division between repositories that I liked and those that I'm interested in and/or following would be really great.
The main idea is to Star the repositories that I'm working on or need to have a quick access and Like the repositories that I liked or I found it interesting

**Comment by becca** [comment]
How do you currently "like" repositories versus show interest versus follow? Can you please expand a bit here on how you define the division you're looking for here and why? cc// @bdresser

**Reply by mikumikudice** [reply]
I think the Star concept it's a mix between "things that I liked and I think that I should demonstrate it staring" and "oh, what a interesting thing, I think I'll Star it to open later and do issues and read the source code". I want to split it
